tests/datetime/test_fluent_setters.py

```diff
@@ -139,18 +139,41 @@ def test_fluid_at_with_transition():
     assert 0 == new.second
 
 
-def test_replace_tzinfo():
-    d = pendulum.datetime(2016, 7, 2, 0, 41, 20)
+def test_replace_tzinfo_dls_off():
```

Comment:
Why dls here an not dst?

Reply:
Oh right - I think I started off calling it "dls" (daylight-savings), but later saw that "dst" was a more common way to name it.

I'll rename to "dst".

Reply:
Fixed with a fixup commit (i.e. I can squash into one commit before merge if you like, unless if you want to just squash+merge)